src/main/java/com/example/account/service/ElasticAccountService.java

```diff
@@ -13,4 +13,6 @@ public interface ElasticAccountService {
     Flux<Account> retrieveByBalanceBetween(int from, int to);
     Flux<Account> retrieveByParams(AccountCriteria criteria, Pageable pageable);
 
+    Flux<Account> retrieveAll();
```

Comment:
@amerkul some inconsistency here

on line 13, and 14 you are using no enter after end of method
but, on line 15 you are separate methods by enter